.NET CLI Template missing namespace

** Description **
A new project created using the HotChocolate.Templates.Server does not build

**To Reproduce**
```powershell
mkdir HotChocolateTemplateIssue
cd .\HotChocolateTemplateIssue\
dotnet new graphql -n HotChocolateTemplateIssue
cd .\HotChocolateTemplateIssue\
dotnet build
```
Produces

```
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 477.31 ms for D:\Source\Scratch\HotChocolateTemplateIssue\HotChocolateTemplateIssue\HotChocolateTemplateIssue.csproj.
Startup.cs(22,22): error CS1061: 'IServiceCollection' does not contain a definition for 'AddDataLoaderRegistry' and no accessible extension method 'AddDataLoaderRegistry' accepting a first argument of type 'IServiceCollection' could be found (are you missing a using directive or an assembly reference?) [D:\Source\Scratch\HotChocolateTemplateIssue\HotChocolateTemplateIssue\HotChocolateTemplateIssue.csproj]
Startup.cs(25,33): error CS0103: The name 'SchemaBuilder' does not exist in the current context [D:\Source\Scratch\HotChocolateTemplateIssue\HotChocolateTemplateIssue\HotChocolateTemplateIssue.csproj]
Startup.cs(25,22): error CS1061: 'IServiceCollection' does not contain a definition for 'AddGraphQL' and no accessible extension method 'AddGraphQL' accepting a first argument of type 'IServiceCollection' could be found (are you missing a using directive or an assembly reference?) [D:\Source\Scratch\HotChocolateTemplateIssue\HotChocolateTemplateIssue\HotChocolateTemplateIssue.csproj]

Build FAILED.

Startup.cs(22,22): error CS1061: 'IServiceCollection' does not contain a definition for 'AddDataLoaderRegistry' and no accessible extension method 'AddDataLoaderRegistry' accepting a first argument of type 'IServiceCollection' could be found (are you missing a using directive or an assembly reference?) [D:\Source\Scratch\HotChocolateTemplateIssue\HotChocolateTemplateIssue\HotChocolateTemplateIssue.csproj]
Startup.cs(25,33): error CS0103: The name 'SchemaBuilder' does not exist in the current context [D:\Source\Scratch\HotChocolateTemplateIssue\HotChocolateTemplateIssue\HotChocolateTemplateIssue.csproj]
Startup.cs(25,22): error CS1061: 'IServiceCollection' does not contain a definition for 'AddGraphQL' and no accessible extension method 'AddGraphQL' accepting a first argument of type 'IServiceCollection' could be found (are you missing a using directive or an assembly reference?) [D:\Source\Scratch\HotChocolateTemplateIssue\HotChocolateTemplateIssue\HotChocolateTemplateIssue.csproj]
    0 Warning(s)
    3 Error(s)
```

**Expected behavior**
The build should be successful

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 10 Pro

**Additional context**
The `Startup.cs` file is missing a using for `HotChocolate`. Inserting the following line at line 4 resolves the issue:

```cs
using HotChocolate;
```

--- a/src/Templates/Server/content/HotChocolate.Server.Template.csproj
+++ b/src/Templates/Server/content/HotChocolate.Server.Template.csproj
@@ -17,13 +17,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="HotChocolate.AspNetCore" Version="10.3.0" />
-    <PackageReference Include="HotChocolate.AspNetCore.Playground" Version="10.3.0" />
-    <PackageReference Include="HotChocolate.AspNetCore.Voyager" Version="10.3.0" />
-    <PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="10.3.0" />
-    <PackageReference Include="HotChocolate.Types.Filters" Version="10.3.0" />
-    <PackageReference Include="HotChocolate.Types.Sorting" Version="10.3.0" />
-    <PackageReference Include="HotChocolate.Subscriptions.InMemory" Version="10.3.0" />
+    <PackageReference Include="HotChocolate.AspNetCore" Version="10.3.2" />
+    <PackageReference Include="HotChocolate.AspNetCore.Playground" Version="10.3.2" />
+    <PackageReference Include="HotChocolate.AspNetCore.Voyager" Version="10.3.2" />
+    <PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="10.3.2" />
+    <PackageReference Include="HotChocolate.Types.Filters" Version="10.3.2" />
+    <PackageReference Include="HotChocolate.Types.Sorting" Version="10.3.2" />
+    <PackageReference Include="HotChocolate.Subscriptions.InMemory" Version="10.3.2" />
   </ItemGroup>
 
 </Project>

--- a/src/Templates/Server/content/Startup.cs
+++ b/src/Templates/Server/content/Startup.cs
@@ -1,9 +1,9 @@
-using HotChocolate;
-using HotChocolate.AspNetCore;
-using HotChocolate.AspNetCore.Voyager;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
+using HotChocolate;
+using HotChocolate.AspNetCore;
+using HotChocolate.AspNetCore.Voyager;
 
 namespace HotChocolate.Server.Template
 {

--- a/src/Templates/Server/content/Startup.cs
+++ b/src/Templates/Server/content/Startup.cs
@@ -1,8 +1,9 @@
+using HotChocolate;
+using HotChocolate.AspNetCore;
+using HotChocolate.AspNetCore.Voyager;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
-using HotChocolate.AspNetCore;
-using HotChocolate.AspNetCore.Voyager;
 
 namespace HotChocolate.Server.Template
 {

